react/week4/github_users_search/src/UserList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import searchContext from "./context";
+import './index.css';
+import UserRepo from "./UserRepo";
+
+function UserList() {
```

Comment:
A functional component: Neat! Using them makes it clear that this component does not have any side effects (e.g. `WillMount`)